README.md

```diff
@@ -19,24 +19,34 @@ Opni currently features log anomaly detection for Kubernetes.
   * Ability to setup & send alerts (slack/email/etc) based on Opni log insights
 
 ----
-#### Try it out!
+### Install Opni in your Kubernetes cluster:
+* Download the `opnictl` binary from the [latest release](https://github.com/rancher/opni/releases/tag/v0.1.2)
```

Comment:
The url `https://github.com/rancher/opni/releases/latest` will redirect to the latest release.

Reply:
Yeah this is intended